Automatic Validation ended with: 
> 2026-03-15T05:27:27.4486088Z ##[error] Manifest Error: Duplicate installer entry found.

(Automated response - build 1235.)

--- a/manifests/d/DCSS/DungeonCrawlStoneSoup/0.34.1/DCSS.DungeonCrawlStoneSoup.installer.yaml
+++ b/manifests/d/DCSS/DungeonCrawlStoneSoup/0.34.1/DCSS.DungeonCrawlStoneSoup.installer.yaml
@@ -7,7 +7,6 @@ InstallerLocale: en-US
 Platform:
 - Windows.Desktop
 InstallerType: nullsoft
-Scope: machine
 InstallModes:
 - interactive
 - silent
@@ -20,11 +19,13 @@ InstallationMetadata:
   DefaultInstallLocation: '%ProgramFiles(x86)%\Crawl'
 Installers:
 - Architecture: x86
+  Scope: machine
   InstallerUrl: https://github.com/crawl/crawl/releases/download/0.34.1/dcss-0.34.1-win32-installer.exe
   InstallerSha256: 9E193B88385760A77CBFC12BCB5ACE873FA22FEBE3AB69C881F26D30A238F52A
   InstallerSwitches:
     Custom: /ALLUSERS
 - Architecture: x86
+  Scope: user
   InstallerUrl: https://github.com/crawl/crawl/releases/download/0.34.1/dcss-0.34.1-win32-installer.exe
   InstallerSha256: 9E193B88385760A77CBFC12BCB5ACE873FA22FEBE3AB69C881F26D30A238F52A
   InstallerSwitches:

